pkgs/top-level/python-packages.nix

```diff
@@ -14585,6 +14587,17 @@ in {
 
   typing = callPackage ../development/python-modules/typing { };
 
+  typing-extensions = buildPythonPackage rec {
```

Comment:
Do you mean there is a conflict with some newer version of nixpkgs?

Reply:
This package simply already exists. However, that's on the master branch only. Anyway, you should make your pull request against master as explained in https://nixos.org/nixpkgs/manual/#chap-submitting-changes

Reply:
Manual says "You can make branch from a commit of your local nixos-version. That will help you to avoid additional local compilations". I use separate nixpkgs tree for this PR and branched from nixos-18.03 of nixpkgs-channels. Is it wrong?

Should I cherry-pick `typing-extensions` commit from master to this PR in order for git-merge to handle it?

Reply:
I cite:

> Rebase you branch against current master.

Someone should fix that typo though :D

Reply:
OK, found this place. Given the cite I made earlier, I should say the manual is somewhat misleading here.